### ISSUE TITLE
Ablation on original training losses?

### ISSUE THREAD

**Issue by DianCh** [PeizeSun/OneNet#28]
Hi! Thanks for releasing such a wonderful work. From the code it looks like the training loss and matching cost are identical, i.e., the original losses used in FCOS, RetinaNet, CenterNet are also replaced with it, which was a bit unclear from the paper. Have you tried using the matching cost only to find out one-2-one matching, but keeping the original losses? Are results for the three detectors **without** o2o matching also obtained with the new training loss (same as the matching cost)?

**Comment by PeizeSun** [comment]
Hi~
We have not tried yet.

**Comment by DianCh** [comment]
Thanks for your reply. When producing results for FCOS, RetinaNet, CenterNet in Table 1 and 2, are the training loss all identical as the matching loss?

**Comment by PeizeSun** [comment]
Yep, the training loss is all identical as the matching loss.
